configs/DAKEFPVF722X8/config.h

```diff
@@ -0,0 +1,130 @@
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        DAKEFPVF722X8
+#define MANUFACTURER_ID   DAKE
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#ifndef USE_CAMERA_CONTROL
+#define USE_CAMERA_CONTROL
+#endif
+
+#define BEEPER_PIN           PC3
+#define MOTOR1_PIN           PC6
+#define MOTOR2_PIN           PC7
+#define MOTOR3_PIN           PC8
+#define MOTOR4_PIN           PC9
+#define MOTOR5_PIN           PA8
+#define MOTOR6_PIN           PA15
+#define MOTOR7_PIN           PB7
+#define MOTOR8_PIN           PA1
+#define LED_STRIP_PIN        PA0
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART4_TX_PIN         PC10
+#define UART5_TX_PIN         PC12
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PC11
+#define UART5_RX_PIN         PD2
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PC14
+#define LED1_PIN             PC15
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI3_SCK_PIN         PB3
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI3_SDI_PIN         PB4
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define SPI3_SDO_PIN         PB5
+#define CAMERA_CONTROL_PIN   PB0
+#define ADC_VBAT_PIN         PC1
+#define ADC_RSSI_PIN         PC2
+#define ADC_CURR_PIN         PC0
+#define PINIO1_PIN           PC13
+#define PINIO2_PIN           PB1
+#define FLASH_CS_PIN         PB6
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PC6 , 2, 0) \
+    TIMER_PIN_MAP( 1, PC7 , 2, 1) \
+    TIMER_PIN_MAP( 2, PC8 , 2, 1) \
+    TIMER_PIN_MAP( 3, PC9 , 2, 0) \
+    TIMER_PIN_MAP( 4, PA8 , 1, 0) \
+    TIMER_PIN_MAP( 5, PA15, 1, 0) \
+    TIMER_PIN_MAP( 6, PB7 , 1, 0) \
+    TIMER_PIN_MAP( 7, PA1 , 1, 0) \
+    TIMER_PIN_MAP( 8, PA0 , 2, 0) \
+    TIMER_PIN_MAP( 9, PB0 , 2, -1)
```

Comment:
Motors split between too many timers. M1-4 on TIM8, but M5-8 are on TIM1, TIM2, and TIM4.